modules/ROOT/pages/change-jdk-config-in-projects.adoc

```diff
@@ -53,21 +53,165 @@ In Studio 7.19.0 and later, your new Mule projects use JDK 17 by default and the
++
+[source,java]
+----
+#
+# List of providers and their preference orders:
+#
+security.provider.1=org.bouncycastle.jcajce.provider.BouncyCastleFipsProvider
+security.provider.2=org.bouncycastle.jsse.provider.BouncyCastleJsseProvider fips:BCFIPS
+# ...
+----
++
+. Save your changes.
+. In Studio, go to your project, right click on its folder, and select *Run configuration*.
+. Go to the *Arguments* tab. In *VM arguments*, add:
++
+[source,text]
+----
+-Dmule.security.model=fips140-2 
+-Dorg.bouncycastle.fips.approved_only=true
+----
++
+. Run your application. The startup readout shows FIPS security model enabled.
++
+[source,text]
+----
+* Mule Runtime and Integration Platform
+* Version: 4.6.21 Build: adffffac
+* MuleSoft, Inc.
+* For more information go to
+* https://www.mulesoft.com/platform/soa/mule-esb-enterprise
+*
+* Server started: 6/4/18 4:06 PM
+* JDK: 1.8.0_172 (mixed mode)
+* JDK properties:
+  - java.vendor = Oracle Corporation
+  - java.vm.name = Java HotSpot(TM) 64-Bit Server VM
+  - java.home = /Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre
+* OS: Mac OS X (10.11, x86_64)
+* Host: your-hostname.local (10.250.0.81)
+* Security model: fips140-2 //<!--1-->
+* Mule services:
+  - api-gateway-contract-service-1.0.0-mule-service.jar
+  - mule-service-http-ee-1.1.1-mule-service.jar
+  - mule-service-oauth-1.1.1-mule-service.jar
+  - mule-service-scheduler-1.1.4-mule-service.jar
+  - mule-service-soap-1.1.2-mule-service.jar
+  - mule-service-weave-ee-2.1.2-mule-service.jar
+* Mule system properties:
+  - mule.security.model = fips140-2
+  - mule.base = /Users/your-user/AnypointStudio/studio-workspace/.mule
+  - mule.home = /Users/your-user/AnypointStudio/studio-workspace/.mule
+----
+[calloutlist]
+.. FIPS 140-2 security model is enabled.
+
+=== Configure FIPS 140-2 in Java 11 and Later
+
+. Download the provider files from the https://www.bouncycastle.org/download/bouncy-castle-java-fips/[BouncyCastle] website:
++
+--
+* Provider: bc-fips-1.0.2.4
+* TLS: bctls-fips-1.0.19.jar
+* PKIX: bcpkix-fips-1.0.7.jar
+--
++
+. Copy the downloaded files to the embedded Mule runtime `.../lib/boot` directory. The path to this directory can vary depending on your Studio and Mule runtime versions.
++
+[source,bash]
+----
+cp bc-fips-1.0.2.4.jar bctls-fips-1.0.19.jar bcpkix-fips-1.0.7.jar /Applications/AnypointStudio.app/Contents/Eclipse/plugins/org.mule.tooling.server.4.9.ee_7.21.0.202507071837/lib/boot
+----
++
+The default path on Windows is `C:\AnypointStudio\plugins\org.mule.tooling.server.4.9.ee_7.21.0.202507071837\lib\boot`.
++
+. Delete the *wrapper.conf* file at `AnypointStudio\plugins\org.mule.tooling.server.4.9.ee_7.21.0.202507071837\mule\conf`.
+. Edit `AnypointStudio\plugins\org.mule.tooling.server.4.9.ee_7.21.0.202507071837\mule\conf\wrapper.conf.template` and uncomment this line by deleting the starting *#* character:
```

Comment:
Add: 

Edit file `/Applications/AnypointStudio.app/Contents/Eclipse/plugins/org.mule.tooling.jdk.macosx.aarch64_1.4.1/Contents/Home/conf/security/java.security` to list only the BouncyCastle security providers: 
```
security.provider.1=org.bouncycastle.jcajce.provider.BouncyCastleFipsProvider
security.provider.2=org.bouncycastle.jsse.provider.BouncyCastleJsseProvider fips:BCFIPS
```

Reply:
OK! Gracias!